schema/proto3/common.proto

```diff
@@ -34,10 +35,11 @@ enum OperationType {
     GET = 2;
     DELETE = 3;
     INFO = 4;
-    FINALIZE = 5;
-    CREATE_UPDATE_SWITCH = 6;
+    FINALIZE = 5; // FIXME: we will try to the whole port operation in the FINALIZE call
+    CREATE_UPDATE_SWITCH = 6; // FIXME: I don't think we need this for OVS?
     CREATE_UPDATE_ROUTER = 7;
```

Comment:
CREATE_UPDATE_SWITCH and CREATE_UPDATE_ROUTER shouldn't be used for OVS scenarios. CREATE_UPDATE_GATEWAY might be. Let us discuss.

Reply:
Yes, we haven't close done on the Router/DVR design yet. But we can have some initial discussions.

Reply:
@haboy52581 These are the fields we discussed.